ocsci/framework.py

```diff
@@ -0,0 +1,144 @@
+import importlib
+import logging
+import os
+import time
+import traceback
+from pprint import pformat
+
+from utility.polarion import post_to_polarion
+from utility.utils import create_unique_test_name, configure_logger, timestamp
+from .enums import TestStatus
+
+log = logging.getLogger(__name__)
+
+
+class TestCase(object):
+    """
+        Generic test case object to contain configuration details from suite.
+        Handles test execution as well as pre and post execution steps
+        for Report Portal and Polarion.
+
+    Attributes
```

Comment:
Args:

Reply:
These are not arguments though, args are outlined in the pydoc for the `__init__`. These are object attributes. Not all are passed to the `__init__` upon instantiation.